### ISSUE TITLE
2021-11-5

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#646]
271Distilling the Knowledge in a Neural Network Hilton NIPS 2014; KD的开山之作，核心思想是使用softed labels去学习class distribution，具体先训练好一个teacher网络，然后将teacher的网的输出结果q作为student网络的目标，训练student网络，使得student网络的结果p接近q。提出的新idea是softmax的变形，引入一个变量T去产生softed labels。soft target 与hard target区别如下图所示。
272Deep mutual learning CVPR 2018; 传统蒸馏模型是从功能强大的大型网络或集成网络转移到结构简单，运行快速的小型网络。本文打破这种预先定义好的“强弱关系”，提出DML，即让一组学生网络在训练过程中相互学习、相互指导，而不是静态的预先定义好教师和学生之间的单向转换通路。
273Distillation-based training for multi-exit architectures ICCV 2019 ; 通常KD在student和teacher network网络模型容量相差较大时，表现较差。Ensemble distribution KD方法可以很好的保证分布diversity，同时多个模型融合的结构较好。大致思路：作者借鉴muti-exit architectures去做ensemble distribution KD，扩展提出新的损失函数。
274Logits(Response)-based Knowledge优缺点总结
2751.简单易于理解，student模型学teacher模型输出的概率分布，相当于给出了类别之间的相似性信息，提供了额外的监督信号，学起来更容易。2.对于输出层，实现简单方便
276Feature-based Knowledge ; 
277WTF is logits? What does logit mean in deep learning? In context of deep learning the logits layer means the layer that feeds in to softmax (or other such normalization). The output of the softmax are the probabilities for the classification task and its input is logits layer. (2021-10-23)
278Backup ssfcdb; done; (2021-10-25)
279Add relative entropy to database; done; (2021-10-25)
